# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/painting/FrameLayerBuilder.h
# Commit: 6f11e76d3822
# Full Hash: 6f11e76d382220103cb9b65b06964429aeeb9010
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-06-15 09:53:16
# Description:
#   Bug 1348503 - Retrieve the FrameLayerBuilder pointer from the LayerManager when we need it. r=tnikkel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D34772
# ==============================================================================

diff -r 41ab55fee001 -r 6f11e76d3822 layout/painting/FrameLayerBuilder.h
--- a/layout/painting/FrameLayerBuilder.h	Fri Jun 14 22:55:27 2019 +0000
+++ b/layout/painting/FrameLayerBuilder.h	Thu Jun 13 17:02:54 2019 +0000
@@ -229,7 +229,6 @@
 
 struct InactiveLayerData {
   RefPtr<layers::BasicLayerManager> mLayerManager;
-  FrameLayerBuilder* mLayerBuilder;
   RefPtr<layers::Layer> mLayer;
   UniquePtr<layers::LayerProperties> mProps;
 
